src/components/CodeInput/readme.md

```diff
@@ -0,0 +1,95 @@
+##### CodeInput base:
+
+```js
+import React, { useEffect, useRef, useState } from 'react';
+import { CodeInput } from 'react-rainbow-components';
+
+function CodeInputBase() {
+    const [code, setCode] = useState();
+    const codeInputRef = useRef();
+
+    useEffect(() => {
+        codeInputRef.current.focus();
+    }, []);
+
+    return (
+        <CodeInput
+            id="codeinput-1"
+            value={code}
+            label="Code"
+            required={true}
+            onChange={(code) => { setCode(code); }}
```

Comment:
you can ```onChange={setCode}```

Reply:
:O, I was only using it for the onClick, onFocus, and onEvent stuff... I see now